install/tools/Makefile.am

```diff
@@ -37,4 +37,6 @@ dist_app_SCRIPTS =		\
 	ipa-custodia-check	\
 	ipa-httpd-kdcproxy	\
 	ipa-pki-retrieve-key	\
+	ipa-httppswd.sh		\
+	ipa-pwdreader.sh	\
```

Comment:
Why did you add two scripts? It looks like an unnecessary complicated to me.

Reply:
Explained in the commit message, one symlinks to the other.